app/car.py

```diff
@@ -0,0 +1,14 @@
+import math
+
+
+class Car:
+    def __init__(self, brand: str, fuel_consumption: float) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    def fuel_cost(self,
+                  cust_loc: list[int],
+                  shop_loc: list[int],
```

Comment:
Do not use abbreviations when you can avoid them!
cust_loc -> customer_location
shop_loc -> shop_location